### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by UjjwalBudha**
completed the task and sent for review

### CODE REVIEW

File: ansible/playbook.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+# - hosts: i-003db8f1004f8f52b
```
Comment (mukes137) [inline ansible/playbook.yml:2]: remove the commented lines.

File: ansible/playbook.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+#   become: true
+#   roles:
+#     - django
+#   vars:
+#     ansible_connection: aws_ssm
+#     ansible_aws_ssm_bucket_name: 8586-terraform-state 
+#     ansible_aws_ssm_profile: default
+#     ansible_aws_ssm_timeout: 300
+
+- hosts: i-090b0d4c36dbcc597
```
Comment (mukes137) [inline ansible/playbook.yml:13]: do not use instance id as hostname.
make it more dynamic in nature.

File: ansible/roles/django/tasks/main.yml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+---
+- name: Install Nginx on Public_Server
+  apt:
+    name: nginx
+    state: present
+
+- name: Configure Nginx reverse proxy
+  template:
+    src: main.j2
+    dest: /etc/nginx/sites-available/reverse_proxy.conf
```
Comment (mukes137) [inline ansible/roles/django/tasks/main.yml:10]: use variables instead of hardcoded path.

File: ansible/roles/django/tasks/main.yml [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+  notify: Restart Nginx
+
+- name: Enable reverse proxy configuration
+  file:
+    src: /etc/nginx/sites-available/reverse_proxy.conf
```
Comment (mukes137) [inline ansible/roles/django/tasks/main.yml:15]: use variables.

File: ansible/roles/django/handlers/main.yml [hunk 5]
```diff
@@ -0,0 +1,5 @@
+---
+- name: Restart Nginx
```
Comment (mukes137) [inline ansible/roles/django/handlers/main.yml:2]: why are you restarting nginx in django task?

File: ansible/roles/nginx/vars/main.yml [hunk 6]
```diff
@@ -0,0 +1,35 @@
+# ---
+# home_dir: /root
```
Comment (mukes137) [inline ansible/roles/nginx/vars/main.yml:2]: remove the commented lines.

**Review by mukes137 — CHANGES_REQUESTED** [review]

File: ansible/playbook.yml (hunk 1 above)
Comment (UjjwalBudha) [reply ansible/playbook.yml]: uncommented this part

File: ansible/roles/django/tasks/main.yml (hunk 3 above)
Comment (UjjwalBudha) [reply ansible/roles/django/tasks/main.yml]: used variables

File: ansible/roles/django/handlers/main.yml (hunk 5 above)
Comment (UjjwalBudha) [reply ansible/roles/django/handlers/main.yml:2]: in my case django is the frontend part and after changing  the reverse proxy conf i have restarted the frontend server which will now interact with the backend server[nginx]

File: ansible/roles/nginx/vars/main.yml (hunk 6 above)
Comment (UjjwalBudha) [reply ansible/roles/nginx/vars/main.yml]: removed the commented parts

File: ansible/roles/django/tasks/main.yml (hunk 4 above)
Comment (UjjwalBudha) [reply ansible/roles/django/tasks/main.yml]: used variables

File: ansible/playbook.yml (hunk 2 above)
Comment (UjjwalBudha) [reply ansible/playbook.yml]: updated it

**Review by mukes137 — APPROVED** [review]

**Pull request merged**